index.js

```diff
@@ -62,17 +74,68 @@ module.exports = {
         */
         'no-loop-func': 'error',
 
+        /*
+            Enforce parenthesis in cases of mixed precedence.
+        */
+        'no-mixed-operators': ['error', {
+            groups: [
+                ['%', '**'],
+                ['%', '+'],
+                ['%', '-'],
+                ['%', '*'],
+                ['%', '/'],
+                ['/', '*'],
+                ['&', '|', '<<', '>>', '>>>'],
+                ['==', '!=', '===', '!=='],
+                ['&&', '||'],
+            ],
+            allowSamePrecedence: true
+        }],
+
         /*
             This rule defends against prototype pollution vulnerabilities that exploitable with
             XSS.
         */
         'no-prototype-builtins': 'error',
 
+        /*
+            Higher-order function composition is preferred for iterative loops. For...in statements
+            can be used when performance is a consideration as long as guard-for-in is enabled.
+            For..of statements can be used when break is required for control flow or the iteration
```

Comment:
My nitpicking mind can't stop seeing that we use three dots in `for...in` but two in `for..of`. Can we use three for both?